config/routes.rb

```diff
@@ -3,5 +3,6 @@
   ActiveAdmin.routes(self)
   devise_for :users
   root "texts#index"
+  resources :challenges, only: [:index]
```

Comment:
後で詳細ページが必要になりますので，追加しておいて下さい。

```suggestion
  resources :challenges, only: [:index, :show]
```